java/arcs/sdk/wasm/WasmMultipleHandles.kt

```diff
@@ -0,0 +1,781 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.sdk.wasm
+
+/** Combined Handle to allow events on two handles to trigger actions. */
+class WasmTupleHandle<T, U>(
```

Comment:
Tuples are by definition multi-argument (0...N), I think you're after "double" here.